docs/contracts/technical-references/taking-and-making-offers/reactive-offer/offer-provision.md

```diff
@@ -194,52 +198,16 @@ If you frequently update your offers, we recommend using a consistent, high `gas
 
 :::
 
-## Provision and offer bounty
-
-<Tabs>
-<TabItem value="soliditiy" label="Solidity" default>
-
-```solidity
-import "src/IMangrove.sol";
-import {MgvStructs} from "src/MgvLib.sol";
-//context 
-IMangrove mgv;
-address outbound_tkn;
-address inbound_tkn;
-uint offer_gasreq;
-(MgvStructs.GlobalPacked global32, MgvStructs.LocalPacked local32) = mgv.config(outbound_tkn, inbound_tkn);
-
-// computing minimal provision to cover an offer requiring `offer_gasreq` gas units 
-uint provision = (offer_gasreq + local32.offer_gasbase()) * global32.gasprice() * 10 ** 9;
-```
-
+## Bounty calculation
 
-</TabItem>
-<TabItem value="ethersjs" label="ethers.js">
-
-```javascript
-const { ethers } = require("ethers");
-let outTkn; // address of outbound token ERC20
-let inbTkn; // address of inbound token ERC20
-let MGV_reader_address; // address of Mangrove reader
-let MGV_reader_abi; // Mangrove Reader contract's abi
-
-const MangroveReader = new ethers.Contract(
-    MGV_reader_address, 
-    MGV_reader_abi, 
-    ethers.provider
-    );
-
-const ofr_gasreq = ethers.parseUnits("1",5); //100,000 gas units
-const provision = await MangroveReader.getProvision(outTkn, inbTkn, ofr_gasreq,0);
-```
-
-</TabItem>
-</Tabs>
+The bounty is paid to the taker **as compensation for spent gas**. It depends on how much gas the offer uses before failing.
+It is calculated with the following formula, based on the provision [previously calculated](./offer-provision.md#provision-calculation):
 
-The bounty paid to the taker if an offer fails depends on how much gas the offer uses before failing. The bounty amount is calculated with the following formula:
+$$\textrm{bounty} = \min(\textrm{offer.provision},(\textrm{gasreq} + \textrm{gasbase}_{\textrm{mgv}}) \times \textrm{gasbase}_{\textrm{mgv}} \times 10^9)$$
 
-$$\textrm{bounty} = \min(\textrm{offer.provision}, 
-  (\textrm{gas\_used} + \textrm{local.offer\_gasbase}) \times \textrm{global.gasprice} \times 10^9)$$
+* $$\textrm{offer.provision}$$ is the [provision amount](./offer-provision.md#balance-adjustment-when-creatingupdating-offers) calculated when the offer was posted.
+* $$\textrm{gasreq}$$ is the `gasreq` amount of gas units required to execute the offer.
```

Comment:
gasreq should be gasused and is the amount of gas units actually used when executing the offer.